src/ctelnet.cpp

```diff
@@ -946,7 +946,7 @@ void cTelnet::processTelnetCommand(const string& command)
 
                 int newVersion = version.toInt();
                 if (mpHost->mServerGUI_Package_version != newVersion) {
-                    QString _smsg = tr("<The server wants to upgrade the GUI to new version '%1'. Uninstalling old version '%2'>")
+                    QString _smsg = tr("[INFO] The server wants to upgrade the GUI to new version '%1'. Uninstalling old version '%2'")
                                             .arg(QString::number(newVersion), QString::number(mpHost->mServerGUI_Package_version));
                     mpHost->mpConsole->print(_smsg.toLatin1().data());
```

Comment:
Use `mpHost->postMessage(_smsg)` to send the message out - and using `QString::toLatin1()` on a translatable message is ***not*** gonna fly :bomb: ...

Reply:
Wait a minute, I didn't change any of that, just the contents of this string!

Reply:
This is no longer in the PR so marking as resolved as it has gone away.